test/holding-marc-test.js

```diff
@@ -14,17 +14,17 @@ describe('Holding Marc Mapping', () => {
       return holdingSerializer.fromMarcJson(holding)
         .then((statements) => new Holding(statements))
         .then((holding) => {
-          assert.equal(holding.literal('nypl:suppressed'), true)
+          assert.strictEqual(holding.literal('nypl:suppressed'), true)
```

Comment:
All instances of `assert.equal` have been replaced with `assert.strictEqual` both for linting and because tests I've written elsewhere have been tripped up by this. Running the test suite only revealed very minor issues with this here though.